notebooks/eval_notebook.ipynb

```diff
@@ -0,0 +1,721 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "c57d4095bdf746a09e82647b0a3aae7c",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading data:   0%|          | 0.00/87.3M [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "f53c28f7321f4ef2804dbfb4695c59b5",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading data:   0%|          | 0.00/87.5M [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "f5c80960e9b147e893d72bc448bd9494",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Downloading data:   0%|          | 0.00/87.3M [00:00<?, ?B/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "913c03dd73a24684974bf7367ff67a5e",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Generating train split:   0%|          | 0/1087583 [00:00<?, ? examples/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "6abe7fc18d824d7f8a035122340dd401",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Generating validation split:   0%|          | 0/10982 [00:00<?, ? examples/s]"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from delphi.eval import constants\n",
+    "\n",
+    "tokenized_corpus_dataset = cast(Dataset, load_dataset(\n",
+    "    constants.tokenized_corpus_dataset,\n",
+    "    split=split\n",
+    "))\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Processing model llama2-100k\n",
+      "Processing model llama2-200k\n",
+      "Processing model llama2-400k\n",
+      "Processing model llama2-800k\n",
+      "Processing model llama2-1.6m\n",
+      "Processing model llama2-3.2m\n",
+      "Processing model llama2-6.4m\n",
+      "Processing model llama2-12.8m\n",
+      "Processing model llama2-25.6m\n"
+     ]
+    }
+   ],
+   "source": [
+    "from delphi.eval.calc_model_group_stats import calc_model_group_stats\n",
+    "\n",
+    "\n",
+    "token_groups = {id:{'selected':id in selected_ids} for id in vocab.values()}\n",
```

Comment:
Ideally, we want to do this calculation *inside* the visualization functions, and change the visualization input from a label to an arbitrary list of token_ids, no?